README.md

```diff
@@ -39,13 +39,11 @@ Jekyll uses the [Liquid template engine](http://liquidmarkup.org/) for templatin
 
 You can use [GFM](https://kramdown.gettalong.org/parser/gfm.html) fenced code blocks for JavaScript; for example:
 
-<pre>
```

Comment:
Based on the description above, this is intentional. It’s an example of how to write code blocks in Jekyll.